lib/core/managers/settings_manager.dart

```diff
@@ -205,6 +205,14 @@ class SettingsManager with ChangeNotifier {
     return _preferencesService.setBool(flag, value: value);
   }
 
+  /// Get the value of [flag]
+  Future<bool> getBool(PreferencesFlag flag) async {
+    // Log the event
+    _analyticsService.logEvent(
+        "${tag}_${EnumToString.convertToString(flag)}", 'getString');
```

Comment:
```suggestion
        "${tag}_${EnumToString.convertToString(flag)}", 'getBool');
```

Reply:
There is a risk this section break when my PR will land because I also added this change 😅